docker/dockerfiles/Dockerfile

```diff
@@ -54,21 +54,25 @@ FROM ubuntu:18.04 as node
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-    build-essential \
-    curl \
-    g++ \
-    python3-pip
+        build-essential \
+        curl \
+        g++ \
+        g++-8 \
+        gcc-8 \
+        python3-pip
```

Comment:
blargh, vscode auto-"fixed" my indents on the last PR, hopefully ok to sneak two little indent fixes in here? 🙏